burst-junit4/src/main/java/com/squareup/burst/BurstRunner.java

```diff
@@ -104,6 +104,6 @@
    */
   private Description describeChildPlain(FrameworkMethod method) {
     return Description.createTestDescription(getTestClass().getJavaClass(),
-        method.getMethod().getName());
+        method.getMethod().getName(), method.getAnnotations());
```

Comment:
This is not required for the fix, but can be useful in future if you'll need information about annotations inside `BurstRunner.filter()` or if JUnit itself will need this information. I can remove it if you want.